agent/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[tool.poetry]
+name = "agent"
+version = "0.1.0"
+description = "Crystal DBA Agent"
+authors = ["Crystal DBA Developers <info@crystaldba.ai>"]
+readme = "README.md"
+
+packages = [
+    { include = "shared", from = "." },
+    { include = "client", from = "." },
+]
+
+[tool.poetry.scripts]
+lint = "scripts:lint"
+check = "scripts:check"
+build = "scripts:build"
+test = "scripts:test"
+list = "scripts:list_commands"
+package_arm64 = "scripts:build_linux_arm64"
+package_amd64 = "scripts:build_linux_amd64"
+package_macos = "scripts:build_macos_arm64"
+
+[tool.poetry.dependencies]
+python = "~3.12"
+griptape = {extras = ["all"], version = "^1.0.1"}
```

Comment:
Is griptape needed in the client?

Reply:
I believe we only use BaseSqlDriver.RowResult

Can we just copy that in? Otherwise it is excessively heavy-weight.